### PR TITLE
Pull Request for navbar-and-hero

### PR DESCRIPTION
**Pull request by HamzaAnwar1998**
navbar created
hero section created
styles applied
media queries applied

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+            </div>
+            <!-- end of rightside -->
+
+        </section>
+        <!-- end of hero section -->
+
+    </div>
+    <!-- end of wrapper -->
+</body>
```
Comment (ghost) [inline index.html]: [REQUIRED] Although I love the design you are implementing :+1: :sunglasses: I have to ask you you to stick to the [chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). As a future web developer, you will have to build UI based on the design specified in the technical requirements, let's get used to it from the scratch :relaxed: 

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| Screenshot from 2022-02-10 18-16-14 | Screenshot from 2022-02-10 18-22-02 |

In this particular case, you need to change the followings:

- [ ] Make sure your social icons appear after `headline` and they are aligned accordingly.
- [ ] Please make sure that you are using the font and colours (background and font) defined for the chosen template
- [ ] Please make sure you are setting background image using the `CSS` background-image property sample code:
```CSS
selector {
background-image: url(image_path)
}
```
More on that you can find [here](https://www.w3schools.com/cssref/pr_background-image.asp)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <nav class="navbar">
+            <div class="bars"><i class="fa-solid fa-bars fa-xl"></i></div>
+            <h6 class="title"><a href="#">Hamza Anwar</a></h6>
+            <div class="links">
+                <a href="#">Portofolio</a>
+                <a href="#">About</a>
+                <a href="#">Contact</a>
+                <a href="#"><i class="fa-solid fa-envelope fa-xl"></i></a>
+            </div>
+        </nav>
```
Comment (ghost) [inline index.html]: [REQUIRED] Please kindly make sure that your `<nav />` element is wrapped within the `<header />` tag, in this way we can make your code-base solid and readable. Once it is done, you can separate 
```HTML 
<h6 class="title"><a href="#">Hamza Anwar</a></h6> 
``` 
from your `<nav />` and place within `<header />`. sample code:

```HTML
<header>
       <h6 class="title"><a href="#">Hamza Anwar</a></h6>
       <nav class="navbar">
            <div class="bars"><i class="fa-solid fa-bars fa-xl"></i></div>
            <div class="links">
                <a href="#">Portofolio</a>
                <a href="#">About</a>
                <a href="#">Contact</a>
                <a href="#"><i class="fa-solid fa-envelope fa-xl"></i></a>
            </div>
        </nav>
</header>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hamza Anwar</title>
+    <!-- css -->
+    <link rel="stylesheet" href="styles.css">
+    <!-- font aweosome -->
+    <script src="https://kit.fontawesome.com/d7195d9855.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- wrapper -->
+    <div class="wrapper">
```
Comment (ghost) [inline index.html]: [REQUIRED] Please prefer [sementic tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) over divs as much as possible to improve the document's `accessibility`. `<main />` can be the optimal option here.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+                <!-- social links -->
+                <div class="social-links">
+                    <span><a href="#"><i class="fa-brands fa-facebook-f fa-xl"></i></a></span>
+                    <span><a href="https://www.instagram.com/humzaanwar74/" target="blank"><i class="fa-brands fa-instagram fa-xl"></i></a></span>
+                    <span><a href="https://www.youtube.com/c/jsSolutions" target="blank"><i class="fa-brands fa-youtube fa-xl"></i></a></span>
+                    <span><a href="https://github.com/HamzaAnwar1998/" target="blank"><i class="fa-brands fa-github fa-xl"></i></a></span>
+                    <span><a href="https://www.linkedin.com/in/hamza-anwar-565563156/" target="blank"><i class="fa-brands fa-linkedin-in fa-xl"></i></a></span>
+                    <span><a href="https://twitter.com/anwarhamza919/" target="blank"><i class="fa-brands fa-twitter fa-xl"></i></a></span>
+                </div>
```
Comment (ghost) [inline index.html]: [REQUIRED] Please kindly use the `ul` tag to group social icons, and `li` per icon; each icon has a link (`a` tag). Please refer to the comment about [Sementics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) to discover why.

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @HamzaAnwar1998 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
**To Highlight** :1st_place_medal: 
- All linter checks are passed :heavy_check_mark: 
- Great commit messages :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (HamzaAnwar1998) [reply index.html]: What do you mean by font and colors defined? Are you asking me to use CSS variables?

Comment (ghost) [reply index.html]: the font and colours (background and font) specific to your template in Figma. In this case, just make sure you have the respective background image and you may consider giving a bit more `line-height` to your `<p />` element, and the last one is you need to work on the layout just to make sure your social icons are presented after headline just like it is int the template. good luck!

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @HamzaAnwar1998 ,  

Your project is complete! Great job working on requested changes 🥇
There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**